drivers/gles2/shaders/canvas.glsl

```diff
@@ -469,6 +480,18 @@ void main() {
 		normal_used = true;
 #endif
 
+		// If larger fvfs are used, final_modulate is passed as an attribute.
+		// we need to read from this in custom fragment shaders or applying in the post step,
+		// rather than using final_modulate directly.
+#if defined(FINAL_MODULATE_ALIAS)
+#undef FINAL_MODULATE_ALIAS
+#endif
+#ifdef USE_ATTRIB_MODULATE
```

Comment:
Forgot to mention, wasn't 100% sure we needed this, as although this is one file to godot, it gets split into 2 for vertex and fragment shader. I don't suppose it does any harm...

Reply:
My thinking exactly.